Docs/sphinx_doc/VerticalSCoord.rst

```diff
@@ -0,0 +1,93 @@
+
+ .. role:: cpp(code)
+    :language: c++
+
+Vertical S-coordinate
+=====================
+
+.. _'ROMS documentation': https://www.myroms.org/wiki/Vertical_S-coordinate
+
+These equations are as given in the 'ROMS documentation'_
+
+ROMSX has a generalized vertical, terrain-following, coordinate system.  Currently, the vertical transformation equation :math:'z=z\left( x,y,\sigma ,t\right) ', is available which can support vertical stretching 1D-functions when several constraints are satisfied.
+
+Transformation Equation
```

Comment:
consider commenting on which transformation this corresponds to in ROMS (or if there's a name for it)